lib/screens/dup_screen_data/special_cases.ex

```diff
@@ -0,0 +1,337 @@
+    current_time_period = DateTime.utc_now() |> Screens.Util.time_period()
+
+    case time_ranges do
+      %{^current_time_period => {lo, hi}} ->
+        headway_message = %{
+          icon: "subway-negative-black",
+          text: [
+            %{color: :green, text: "GREEN LINE"},
+            %{special: :break},
+            "every",
+            %{format: :bold, text: "#{lo}-#{hi}"},
+            "minutes"
+          ]
+        }
+
+        {:ok, partial_alert_text, headway_message}
+
+      _ ->
+        {:ok, partial_alert_text}
+    end
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, routes: ["Green-B"], headsign: "Boston College"},
+         %{effect: :shuttle, region: :boundary, routes: ["Green-C"], headsign: "Cleveland Circle"}
+       ]) do
+    kenmore_bc_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, headsign: "BC/Clev. Circ."}
+       ]) do
+    kenmore_bc_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, routes: ["Green-B"], headsign: "Boston College"},
+         %{effect: :shuttle, region: :boundary, routes: ["Green-D"], headsign: "Riverside"}
+       ]) do
+    kenmore_bd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, headsign: "BC/Riverside"}
+       ]) do
+    kenmore_bd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{
+           effect: :shuttle,
+           region: :boundary,
+           routes: ["Green-C"],
+           headsign: "Cleveland Circle"
+         },
+         %{effect: :shuttle, region: :boundary, routes: ["Green-D"], headsign: "Riverside"}
+       ]) do
+    kenmore_cd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, headsign: "Clev. Circ./Riverside"}
+       ]) do
+    kenmore_cd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, routes: ["Green-B"], headsign: "Boston College"},
+         %{
+           effect: :shuttle,
+           region: :boundary,
+           routes: ["Green-C"],
+           headsign: "Cleveland Circle"
+         },
+         %{effect: :shuttle, region: :boundary, routes: ["Green-D"], headsign: "Riverside"}
+       ]) do
+    kenmore_bcd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text([
+         %{effect: :shuttle, region: :boundary, headsign: {:adj, "westbound"}}
+       ]) do
+    kenmore_bcd_fullscreen_alert_text()
+  end
+
+  defp kenmore_fullscreen_alert_text(_), do: nil
+
+  defp kenmore_bc_fullscreen_alert_text do
+    [
+      "No",
+      %{icon: :green_b},
+      %{format: :bold, text: "Boston Coll"},
+      "or",
+      %{icon: :green_c},
+      %{format: :bold, text: "Cleveland Cir"},
```

Comment:
If I remember correctly, you might need an explicit line break (`%{special: :break}`) to keep these two parts on the same line. Feel free to ignore if you already checked all of these cases.